varfish/vueapp/package.json

```diff
@@ -6,11 +6,14 @@
     "serve": "vue-cli-service serve",
```

Comment:
Or is this only for storybook? That's OK, I guess.

Reply:
In the end I didn't use storybook.